pkgs/development/python-modules/meteocalc/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "meteocalc";
+  version = "1.1.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-XfWiQXKJuEIjgNvXPYdGkrMbigVP615jDCC6CUxZfzk=";
+  };
+
+  # Tests are not shipped
```

Comment:
https://github.com/malexer/meteocalc/tree/master/tests ?

Reply:
Switched to GitHub for source. Thanks for the feedback.